packages/components/src/components/telekom/logo/logo.tsx

```diff
@@ -46,6 +46,9 @@ export class Logo {
   @Prop() styles?: string;
   @Prop() focusable: boolean = true;
   @Prop() scrollIntoViewOnFocus: boolean = false;
+  /** (optional) Hide all logo related titles */
+  @Prop() hideTitle?: boolean = false;
```

Comment:
hi @marvinLaubenstein thanks a lot for the PR! I think it would make sense to add this prop to `app-shell` and `app-header` and pipe it down to the logo so the users of the header can also benefit from this. Also, let's call it `logoHideTitle` everywhere.

Reply:
Sounds like a very good idea, I change that.